.github/workflows/main.yaml

```diff
@@ -19,6 +19,6 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: install
-        run: npm install
+        run: npm install -f
```

Comment:
this really sucks

Roam is on React 17, but tldraw is on react 18. so we have to force install dependencies each time for now since they are technically incompatible